landingai/pipeline/frameset.py

```diff
@@ -141,13 +142,16 @@ def from_array(cls, array: np.ndarray, is_bgr: bool = True) -> "Frame":
         im = Image.fromarray(array)
         return cls(image=im)
 
-    def run_predict(self, predictor: Predictor) -> "Frame":
+    def run_predict(self, predictor: Predictor, reuse_session: bool = False) -> "Frame":
```

Comment:
Should this `reuse_session` be True by default, to keep the previous behavior?

Reply:
No, if you are using cloud inference and not calling concurrently, a single session will maintain the connection opened and save SSL negotiation time (i.e. 100s of ms per call)...  A future improvement would be to maintain a pool of connections.